easybuild/easyconfigs/b/BerkeleyGW/BerkeleyGW-1.0.6-intel-2016.02-GCC-4.9.eb

```diff
@@ -0,0 +1,35 @@
+easyblock = 'MakeCp'
+
+name = 'BerkeleyGW'
+version = "1.0.6"
+
+homepage = 'http://www.berkeleygw.org'
+description = """The BerkeleyGW Package is a set of computer codes that calculates the quasiparticle
+                 properties and the optical responses of a large variety of materials from bulk
+                 periodic crystals to nanostructures such as slabs, wires and molecules."""
+
+toolchain = {'name': 'intel', 'version': '2016.02-GCC-4.9'}
+toolchainopts = {'usempi': True}
+
+source_urls = ['http://www.berkeleygw.org/releases/']
+sources = ['BGW-%(version)s.tar.gz']
+checksums = ['8740562da41e87a62eb8d457675e4bfd']
+
+patches = ['BerkeleyGW_ignore_archmk.patch']
+
+files_to_copy = ['*']
+
+prebuildopts = 'COMPFLAG=-DINTEL PARAFLAG=-DMPI MATHFLAG=-DUSESCALAPACK FCPP="cpp -ansi" F90free="$MPIF90 -free" '
+prebuildopts += 'LINK="$MPIF90" FOPTS="$FFLAGS" MOD_OPT="-module " INCFLAG="-I" C_PARAFLAG=-DPARA CC_COMP="$MPICXX" '
```

Comment:
Well, I'd stick to using `generic.mpi.linux.mk` rather than relying on an example `.mk` file that happens to specify some things that match the toolchain being used (since that may change)

also, it will limit the changes required when using a totally different toolchain (for example via `--try-toolchain=foss,...`)